packages/next/src/server/app-render/app-render.tsx

```diff
@@ -2112,12 +2116,14 @@ async function renderToStream(
         {
           ReactDOMServer: require('react-dom/server.edge'),
           element: (
-            <AppWithoutContext
-              reactServerStream={errorServerStream}
-              preinitScripts={errorPreinitScripts}
-              clientReferenceManifest={clientReferenceManifest}
-              nonce={ctx.nonce}
-            />
+            <ServerInsertedHTMLProvider>
```

Comment:
We'll always need `ServerInsertedHTMLProvider` to ensure metadata is inserted in SSR, shall we rename the AppWithoutContext to sth else? @gnoff

Reply:
Yeah not sure why it has that name anymore but let’s figure out a better name